packages/plugin-ext/src/main/browser/preference-registry-main.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2018 Red Hat, Inc. and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import {
+    PreferenceService,
+    PreferenceServiceImpl,
+    PreferenceScope
+} from '@theia/core/lib/browser/preferences';
+import { interfaces } from 'inversify';
+import {
+    MAIN_RPC_CONTEXT,
+    PreferenceRegistryExt,
+    PreferenceRegistryMain,
+} from '../../api/plugin-api';
+import { RPCProtocol } from '../../api/rpc-protocol';
+import { ConfigurationTarget } from '../../plugin/types-impl';
+
+export class PreferenceRegistryMainImpl implements PreferenceRegistryMain {
+    private proxy: PreferenceRegistryExt;
+    private preferenceService: PreferenceService;
+    private preferenceServiceImpl: PreferenceServiceImpl;
```

Comment:
This variable uses only inside constructor. So maybe be Should this constant has locale scope?

Reply:
It makes sense, I'll fix it.